gst/nnstreamer/tensor_common.c

```diff
@@ -30,6 +30,7 @@
  * @brief String representations for each tensor element type.
  */
 static const gchar *tensor_element_typename[] = {
+  [_NNS_NONE] = NULL,
```

Comment:
is it better to set "invalid" or "unknown"?  not null string.

Reply:
Right, It will be updated.